sky/data/data_utils.py

```diff
@@ -225,3 +328,123 @@ def run_upload_cli(command: str, access_denied_message: str, bucket_name: str):
                 raise exceptions.StorageUploadError(
                     f'Upload to bucket failed for store {bucket_name}. '
                     'Please check the logs.')
+
+
+def store_rclone_config(bucket_name: str,
+                        region: str,
+                        apply_changes: bool = True):
+    """creates a configuration files for rclone - used for
+    bucket syncing and mounting """
+    # pylint: disable=import-outside-toplevel
+    import textwrap
+
+    rclone_file = os.path.expanduser(RCLONE_CONFIG_PATH)
```

Comment:
```suggestion
    rclone_config_path = os.path.expanduser(RCLONE_CONFIG_PATH)
```
nit: trying to be more informative

Reply:
NP. Applied similar changes to `delete_rclone_bucket_profile()` and `_remove_bucket_profile_rclone()` as well.